acados/ocp_nlp/ocp_nlp_cost_external.c

```diff
@@ -107,6 +107,14 @@ void ocp_nlp_cost_external_dims_set(void *config_, void *dims_, const char *fiel
 
 
 
+void ocp_nlp_cost_external_dims_get(void *config_, void *dims_, const char *field, int* value)
+{
+        printf("error: ocp_nlp_cost_external_dims_get: attempt to get dimensions of non-existing field %s\n", field);
+        exit(1);
+}
+
+
```

Comment:
almost ;)

just set also the `config->dims_get = &ocp_nlp_cost_external_dims_get` and we are good :)

Reply:
oh god, sorry... 1ccdc